src/js/session_report.js

```diff
@@ -20,6 +20,8 @@ export class SessionReport {
         this._preTalkingTimeMillis = null;
         this._talkingTimeMillis = null;
         this._iceConnectionsLost = 0;
+        this._iceConnectionsFailed = null;
```

Comment:
Why not initialize this to 0 instead of null just like iceConnectionLost?

Reply:
iceConnectionsLost is a number that the ice connection can lost and recover multiple times, but the two new fields are Boolean.
iceConnectionsFailed is supported only in Firefox and is undefined in Chrome. Chrome is planning to support it with date unknow
peerConnectionFailed is only supported by Chrome and will be undefined in Firefox. It would be deprecated once Chrome support iceConnectionsFailed. I did not set them to be false by default so that I can track if those browser still support them as expected